projects/optic/README.md

```diff
@@ -55,50 +55,11 @@ Additional options:
 
 ## Setup Optic in CI
 
-Optic helps you review OpenAPI specs and enforce API standards. Sign up on [app.useoptic.com](https://app.useoptic.com) to get a token and get started.
+Optic helps you review OpenAPI specs and enforce API standards. Sign up on [app.useoptic.com](https://app.useoptic.com).
 
-### GitHub Actions
```

Comment:
Another driveby cleanup - github actions (and the context generators) are no longer used